cmd/wego-server/app/app.go

```diff
@@ -0,0 +1,61 @@
+package app
+
+import (
+	"context"
+	"fmt"
+	"net/http"
+
+	"github.com/grpc-ecosystem/grpc-gateway/v2/runtime"
+	log "github.com/sirupsen/logrus"
+	"github.com/spf13/cobra"
+	pb "github.com/weaveworks/weave-gitops/pkg/api/applications"
+	"github.com/weaveworks/weave-gitops/pkg/server"
+)
+
+func NewAPIServerCommand() *cobra.Command {
+	cmd := &cobra.Command{
```

Comment:
Tiny suggestion would be to move this to the cmd folder for consistency and have all commands in the same place.